### PR TITLE
Improves Bounty Prices and Availability

### PR DESCRIPTION
**Pull request by atteria**
## About The Pull Request
Quadruples bounty respawns ~~and doubles bounty prices~~.

## Why It's Good For The Game
By popular and host request.

## Changelog
:cl:
balance: Quadruples bounty availability
/:cl:

### CODE REVIEW

File: code/modules/bounty/bounty.dm [hunk 1]
```diff
@@ -56,8 +56,8 @@
 	var/cash_max = 0
 
 	//reward stuff
-	var/department_reward = 100 		// In credits.
-	var/individual_reward = 40		// In credits
+	var/department_reward = 200 		// In credits.
+	var/individual_reward = 80		// In credits
```
Comment (genessee-forgot-his-password-again) [inline code/modules/bounty/bounty.dm:60]: These two lines don't actually change anything. They're just dummy numbers for the parent bounty - you'll need to manually change the prices for bounties on their respective pages to see any change.

Comment (genessee-forgot-his-password-again) [inline code/modules/bounty/bounty.dm:60]: forgot to mark as requested change 😳 

feel free to resolve this convo when the above convo is fixed

**Review by genessee-forgot-his-password-again — CHANGES_REQUESTED** [review]

File: code/modules/bounty/bounty.dm (hunk 1 above)
Comment (atteria) [reply code/modules/bounty/bounty.dm:60]: This is what I get for just ctrl+F'ing
We could use some variable as a coefficient for that as a balancing aid, but that would be for another PR.

**Review by genessee-forgot-his-password-again — APPROVED** [review]
Looks good to me!

**Pull request merged**